metricbeat/module/windows/perfmon/_meta/docs.asciidoc

```diff
@@ -0,0 +1,26 @@
+=== windows perfmon MetricSet
+
+This is the perfmon metricset of the module windows.
+
+[float]
+=== Features and configuration
+
+Metricset to collect performance counters.
+Example configuration:
+```
+- module: windows
+  metricsets: ["perfmon"]
+  enabled: true
+  period: 10s
+  perfmon.counters:
+    - group: "processor"  
+      collectors:
+        - alias: "processor_perfomance"
```

Comment:
>Do we need the "group" part, or could we just let the user define processor. processor_perfomance 

I need the group param to get the name. See [here](Do we need the "group" part, or could we just let the user define processor. processor_perfomance) `v.Name`. Or is there an easier way to extract value from config file?